ios/Sources/AboutFeature/AboutScreen.swift

```diff
@@ -31,55 +31,7 @@ public struct AboutScreen: View {
-                            ScrollView(.vertical) {
-                                switch viewStore.selectedType {
-                                case .staff:
-                                    LazyVStack(alignment: .leading, spacing: 24) {
-                                        ForEach(viewStore.staffs) { staff in
-                                            StaffCell(staff: staff) { staff in
-                                                viewStore.send(.tapStaff(staff))
-                                            }
-                                        }
-                                    }
-                                    .padding(.top, 20)
-                                case .contributor:
-                                    LazyVGrid(
-                                        columns: Array(repeating: .init(), count: 3),
-                                        spacing: 40
-                                    ) {
-                                        ForEach(viewStore.contributors) { contributor in
-                                            ContributorCell(contributor: contributor) { contributor in
-                                                viewStore.send(.tapContributor(contributor))
-                                            }
-                                        }
-                                    }
-                                    .listStyle(PlainListStyle())
-                                    .padding(.top, 20)
-                                }
-                            }
-                            .toolbar {
-                                ToolbarItem(placement: .principal) {
-                                    Picker(
-                                        "",
-                                        selection:
-                                            viewStore.binding(
-                                                get: { $0.selectedType },
-                                                send: { .selectedPicker($0) }
-                                            )
-                                    ) {
-                                        ForEach(SelectedType.allCases, id: \.self) { (type) in
-                                            Text(type.title).tag(type)
-                                        }
-                                    }
-                                    .frame(width: geometry.size.width - 32, height: nil, alignment: .center)
-                                    .pickerStyle(SegmentedPickerStyle())
-                                }
-                            }
-                            .background(AssetColor.Background.primary.color.ignoresSafeArea())
-                            .navigationBarTitleDisplayMode(.inline)
-                            .onAppear {
-                                viewStore.send(.refresh)
-                            }
+                            scrollView(geometry: geometry, viewStore: viewStore)
```

Comment:
It is because I get the following error message: 'The compiler is unable to type-check this expression in reasonable time; try breaking up the expression into distinct sub-expressions'

Reply:
In my environment, this is not occurred. 
Can you re-check about this?

Reply:
This is not occurred if I recheck about this 💦
Thanks for your advise.